### PR TITLE
Improve CRD comments

### PR DESCRIPTION
**Pull request by yanjunz97**

### CODE REVIEW

File: build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfos.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ spec:
             description: |-
               PrivateTGWIPCIDRs is a list of CIDR strings. Each CIDR is a contiguous IP address
               spaces represented by network address and prefix length. The visibility of the
-              IPBlocks is PrivateTWG. Only IPBlocks in default project will be included.
+              IPBlocks is Private Transit Gateway. Only IPBlocks in default project will be included.
```
Comment (dantingl) [inline build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfos.yaml:51]: I think it's more about the typo, use `PrivateTGW` as it's the type in NSX API

Comment (yanjunz97) [reply build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfos.yaml]: Updated, thanks!

**Comment by yanjunz97** [comment]
Removed the dhcp pointer part as discussed offline.

File: build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfos.yaml (hunk 1 above)
Comment (fstrudel) [reply build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfos.yaml]: No it should be plain English in the description ideally.

File: build/yaml/crd/vpc/crd.nsx.vmware.com_vpcnetworkconfigurations.yaml [hunk 2]
```diff
@@ -111,8 +111,8 @@ spec:
                   type: object
                 type: array
               vpcs:
-                description: VPCs describes VPC info, now it includes lb Subnet info
-                  which are needed for AKO.
+                description: VPCs describes VPC info, now it includes Load Balancer
+                  Subnet info which are needed for AKO.
```
Comment (fstrudel) [inline build/yaml/crd/vpc/crd.nsx.vmware.com_vpcnetworkconfigurations.yaml:115]: You might want to expand what AKO means like `... which are needed for the Avi Kubernetes Operator (AKO)`.

**Review by fstrudel — APPROVED** [review]

**Review by jianjuns — APPROVED** [review]
Nit: crd -> CRD in the commit description.

File: build/yaml/crd/vpc/crd.nsx.vmware.com_vpcnetworkconfigurations.yaml (hunk 2 above)
Comment (yanjunz97) [reply build/yaml/crd/vpc/crd.nsx.vmware.com_vpcnetworkconfigurations.yaml]: Updated, thanks

**Review by dantingl — APPROVED** [review]

**Review by jianjuns — APPROVED** [review]

**Pull request merged**